src/lib/core/toast.ts

```diff
@@ -47,7 +47,8 @@ const createHandler =
 		return toast.id;
 	};
 
-const toast = <T extends Record<string, unknown> = Record<string, unknown>>(
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const toast = <T extends Record<string, any> = Record<string, any>>(
```

Comment:
The example using the `RichContent` component gave type errors when this used `unknown`. Using `any` here doesn't seem like a problem as the props to `RichContent` were still inferring correctly.